backend/init/init.go

```diff
@@ -38,6 +39,7 @@ func init() {
 		"consul": func() backend.Backend { return backendconsul.New() },
 		"inmem":  func() backend.Backend { return backendinmem.New() },
 		"s3":     func() backend.Backend { return backendS3.New() },
+		"azure":  func() backend.Backend { return backendAzure.New() },
```

Comment:
Good point. @jbardin if we did rename this, would that be likely to be painful, or should we just leave this as-is?

(I raise this as it confused me when I was starting out with Terraform, so I wonder if others have had the same issue)